### PR TITLE
Dndefine

### PR DESCRIPTION
**Pull request by sgryschuk-va**
@gholtslander-va something like this

### CODE REVIEW

**Comment by gholtslander-va** [comment]
Yeah, this would be cool!

**Comment by sgryschuk-va** [comment]
@gholtslander-va I'm not sure how you set up the tokens and what not, but I think this is ready to go up other than that

**Comment by gholtslander-va** [comment]
I make an Incoming Webhook in the Slack org in question, then come back to the code and add it to the list, haha. I can do that 👍

File: src/app/responders/common/dndefine.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+        logging.info('Searching checking dict for term: %s', term)
+        if term.lower() == "list":
+            return self._format_response(term, '\n'.join(DEFINITIONS.iterkeys())) 
+        return self._format_response(term, DEFINITIONS.get(term.lower()))
+
+    @staticmethod
+    def _format_response(term, definition):
+        response = ''
+
+        response += '*{}*\n\n'.format(term)
```
Comment (gholtslander-va) [inline src/app/responders/common/dndefine.py]: This is just waiting to blow up when some yokel tries a unicode character in their term "for fun"

**Review by ghostlandr — APPROVED** [review]

**Pull request merged**